app/views/helpers/rounding_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class RoundingHelper
+  def self.round_decimal_part(num, decimal_sig_figs = 6)
+    return BigDecimal("%.#{decimal_sig_figs}g" % num).to_s if num.abs < 1
+
+    rounded_decimal_part = BigDecimal(num.to_s).frac.round(decimal_sig_figs)
```

Comment:
can you comment on why this is required? 🤔  this feels very convoluted

Reply:
If `rounded_decimal_part` is zero, we need to call `rounded_number.to_i.to_s` instead of just `rounded_number.to_s`.

In this example (which would fail otherwise):

```
expect(helper.round_decimal_part(123.000000000456789)).to eq("123")
```

we want the result to equal `123` instead of `123.0`

Reply:
Okay, I would argue that the following code is cleaner:

```
rounded = BigDecimal(num.to_s).round(decimal_sig_figs)
rounded.frac.zero? ? rounded.to_i.to_s : rounded.to_s
```

(e.g. I find the rounding of the fractional part confusing, but maybe that's just me)

Reply:
I pushed the changes